src/trackedit/internal/au3/au3interaction.h

```diff
@@ -63,11 +63,23 @@ class Au3Interaction : public ITrackeditInteraction
     void newLabelTrack() override;
     void deleteTracks(const TrackIdList& trackIds) override;
     void duplicateTracks(const TrackIdList& trackIds) override;
+    void moveTracksUp(const TrackIdList& trackIds) override;
+    void moveTracksDown(const TrackIdList& trackIds) override;
+    void moveTracksToTop(const TrackIdList& trackIds) override;
+    void moveTracksToBottom(const TrackIdList& trackIds) override;
+    void moveTracksTo(const TrackIdList& trackIds, int to) override;
 
     void undo() override;
     void redo() override;
 
 private:
+    enum class TrackMoveDir {
```

Comment:
It's better not to abbreviate but to use Direction (confused with the directory)

Reply:
Agree